airbyte-webapp/src/components/connection/CatalogTree/next/BulkEditPanel.tsx

```diff
@@ -16,21 +15,28 @@ import { useBulkEditService } from "hooks/services/BulkEdit/BulkEditService";
 import { useConnectionFormService } from "hooks/services/ConnectionForm/ConnectionFormService";
 import { SUPPORTED_MODES } from "views/Connection/ConnectionForm/formConfig";
 
-import { pathDisplayName, PathPopout } from "../PathPopout";
+import { pathDisplayName } from "../PathPopout";
 import { HeaderCell } from "../styles";
-import { SyncSettingsDropdown } from "../SyncSettingsDropdown";
 import { flatten, getPathType } from "../utils";
+import styles from "./BulkEditPanel.module.scss";
+import { StreamPathSelect } from "./StreamPathSelect";
+import { SyncModeSelect } from "./SyncModeSelect";
 
-const SchemaHeader = styled(Header)`
+interface SchemaHeaderProps {
+  isActive: boolean;
+}
+
+const SchemaHeader = styled(Header)<SchemaHeaderProps>`
```

Comment:
This still exists over SCSS because Header is a styled component?

Reply:
yes